app/controllers/sprints_controller.rb

```diff
@@ -59,6 +59,19 @@ def create
         raise 'Fail to update project with product backlog' unless @project.save!
       end
       flash[:notice] = l(:notice_successful_create)
+      pbi_params = {"issue"=>{"tracker_id"=> Tracker.find_by_name('Product Backlog').id, 
+                              "sprint_id"=> @sprint.id, 
+                              "subject"=>"#{@sprint.name} pbi", 
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.